samples/Beyer2024/create_paper_data.py

```diff
@@ -51,7 +52,7 @@
         raise RuntimeError()
     pH_range = np.linspace(2, 12, num=21)
     for pH in pH_range:
-        run_command=["python3", script_path, "--sequence", sequence, "--pH", str(pH), "--mode", mode]
+        run_command=[sys.executable, script_path, "--sequence", sequence, "--pH", str(pH), "--mode", mode]
```

Comment:
Just in case we run on a platform where `python` is used instead of `python3`. Quite rare, but can happen depending on how a virtual environment is configured, or how Python gets packaged on macOS.